scripts/add-module-exports.js

```diff
@@ -0,0 +1,47 @@
+// "add-module-exports"
```

Comment:
For the record I would still like to see https://phabricator.babeljs.io/T6958 happen